### PR TITLE
Skip account if trove is closes or account hasn't open it

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/liquity/trove.py [hunk 1]
```diff
@@ -236,6 +236,9 @@ def get_positions(
             if status is True:
                 try:
                     trove_info = contract.decode(result, 'Troves', arguments=[addresses[idx]])
+                    trove_is_active = bool(trove_info[3])  # pylint: disable=unsubscriptable-object
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/liquity/trove.py:239]: What is trove_info[3] supposed to be? I don't think the bool converter can raise in anyway but just wondering

File: rotkehlchen/chain/ethereum/modules/liquity/trove.py [hunk 2]
```diff
@@ -236,6 +236,9 @@ def get_positions(
...
+                    if not trove_is_active:
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/liquity/trove.py:240]: nitpick:
```suggestion
                    if trove_is_active is False
```

**Review by LefterisJP — COMMENTED** [review]
LGTM

File: rotkehlchen/chain/ethereum/modules/liquity/trove.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/modules/liquity/trove.py:239]: is either `0` or `1`

File: rotkehlchen/chain/ethereum/modules/liquity/trove.py (hunk 2 above)
Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/liquity/trove.py:240]: please do that in another pr. Too small to need  a new commit/run tests again.

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**